mllib/src/main/scala/org/apache/spark/ml/feature/MinHashLSH.scala

```diff
@@ -70,19 +70,46 @@ class MinHashLSHModel private[ml](
 
   @Since("2.1.0")
```

Comment:
`keyDistance` was not exposed to the end users, is this since annotation needed?

Reply:
I suppose it's protected so a little more visible to callers. I wouldn't remove it just for its own sake.